src/app/views/query/form/query-form.component.js

```diff
@@ -280,52 +253,24 @@ class QueryFormController {
     }
   }
 
-  specificConfigCall() {
-    let specificName;
-    if (this.families.length === 1) {
-      specificName = this.families[0];
-    } else if (this.singleProject) {
-      specificName = this.individualProjects[0].projectConfiguration.name;
-    }
-    const firstMatchingConfig = this.projects.find(
-      p => p.projectConfiguration.name === specificName,
+  identifySpecificConfig() {
+    const specificConfigName = this.singleProject
+      ? this.individualProjects[0].projectConfiguration.name
+      : this.families[0];
+    this.matchingProjectForConfigurationRetrieval = this.projects.find(
```

Comment:
don't set this on the component b/c it's only needed in the next call. Just pass it in as a param